Jenkinsfile.release

```diff
@@ -1,23 +1,31 @@
 #!/usr/bin/env groovy
 
 ansiColor('xterm') {
-  node('JenkinsMarathonCI-Debian8-2017-10-23') {
+  node('JenkinsMarathonCI-Debian8-2017-10-06') {
 
     properties([
       parameters([
-        string(name: 'gitsha',
-          defaultValue: '',
-          description: 'Git Commit SHA to Build and Release (ex. 167d7fb86 or 167)'
+        string(name: 'ref',
+          defaultValue: 'origin/releases/1.5',
+          description: 'Git Commit SHA or Branch to Build and Release (ex. 167d7fb86 or 167)'
         ),
         string(name: 'version',
           defaultValue: '',
-          description: 'Community Release Version (ex. v1.5.2)'
+          description: 'Version that should be released. You will need to tag and push a tag after this job succeeds. MUST start with a number. If unspecified, version is inferred from ref.'
```

Comment:
Do we want to be extra clever and infer the version?

Reply:
No :)

Reply:
fail loud and proud.

Reply:
Oh, I see what you mean now, you're asking if we want to allow the version to be inferred.

Yeah, we should probably make it required. I don't see of a valid reason to allow it to be inferred.

Reply:
it should be required and checked